2024-01-04 18:30:47 ERROR Unable to initialize WebDriver
2024-01-04 18:30:47 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 82, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 414, in element_is_visible
    return Wait(self.browser, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcom-CcJ6hTD8-py3.11/lib/python3.11/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x5606d11bcf83 <unknown>
#1 0x5606d0e75cf7 <unknown>
#2 0x5606d0ec599e <unknown>
#3 0x5606d0ec5aa1 <unknown>
#4 0x5606d0f10d64 <unknown>
#5 0x5606d0eef0dd <unknown>
#6 0x5606d0f0e006 <unknown>
#7 0x5606d0eeee53 <unknown>
#8 0x5606d0eb6dd4 <unknown>
#9 0x5606d0eb81de <unknown>
#10 0x5606d1181531 <unknown>
#11 0x5606d1185455 <unknown>
#12 0x5606d116df55 <unknown>
#13 0x5606d11860ef <unknown>
#14 0x5606d115199f <unknown>
#15 0x5606d11aa008 <unknown>
#16 0x5606d11aa1d7 <unknown>
#17 0x5606d11bc124 <unknown>
#18 0x7f6884c94ac3 <unknown>

2024-01-04 18:30:51 ERROR Unable to initialize WebDriver
2024-01-04 18:30:51 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 82, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 414, in element_is_visible
    return Wait(self.browser, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcom-CcJ6hTD8-py3.11/lib/python3.11/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x5606d11bcf83 <unknown>
#1 0x5606d0e75cf7 <unknown>
#2 0x5606d0ec599e <unknown>
#3 0x5606d0ec5aa1 <unknown>
#4 0x5606d0f10d64 <unknown>
#5 0x5606d0eef0dd <unknown>
#6 0x5606d0f0e006 <unknown>
#7 0x5606d0eeee53 <unknown>
#8 0x5606d0eb6dd4 <unknown>
#9 0x5606d0eb81de <unknown>
#10 0x5606d1181531 <unknown>
#11 0x5606d1185455 <unknown>
#12 0x5606d116df55 <unknown>
#13 0x5606d11860ef <unknown>
#14 0x5606d115199f <unknown>
#15 0x5606d11aa008 <unknown>
#16 0x5606d11aa1d7 <unknown>
#17 0x5606d11bc124 <unknown>
#18 0x7f6884c94ac3 <unknown>